### PR TITLE
7111 housekeeping: gulpfile tweaks and move rev-manifest to /resources

### PR DESCRIPTION
**Pull request by gmarinov**
this PR

- moves rev-manifest.json to /resources
- adds filesystem polling to `gulp watch` (less aggressive polling for low CPU usage)
- adds 'clean-all' gulp task

shouldn't change any functionality or looks otherwise

thanks @benolot

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: gulpfile.js [hunk 1]
```diff
@@ -123,47 +120,51 @@ gulp.task('images', gulp.series('images:clean', function() {
 
 // ------
 
-gulp.task('rev', gulp.series(gulp.parallel('sass', 'images', 'js'), function() {
+gulp.task('rev:clean', () => del([manifestPath.join('/')]));
+
+gulp.task('rev', gulp.series('rev:clean', gulp.parallel('sass', 'images', 'js'), function() {
     return gulp.src([staticPathDist + '/**/*', '!' + staticPathDist + '/**/rev-manifest.json'])
```
Comment (benolot) [inline gulpfile.js]: Does this need updating to use the new variable too?

Comment (gmarinov) [reply gulpfile.js]: hmm.. moved to `rev:clean` in 008bb6e

Comment (gmarinov) [reply gulpfile.js]: hmm.. moved to `rev:clean` in 008bb6e

**Pull request merged**